DuckDuckGo/MainViewController.swift

```diff
@@ -568,3 +577,26 @@ extension MainViewController: BookmarksDelegate {
     }
 }
 
+// This preloads a webview so that the datastore can then receive cookies, otherwise nothing happens until subsequent webviews are shown
```

Comment:
Apple is making us do some interesting workarounds today! There's a comment on the 9th of May in your link that mentions a workaround by accessing the data store. Is that something we can do instead of this?

Reply:
I tried that too, didn’t work for me :(